docs/plugins/verifier/cosign.md

```diff
@@ -54,13 +54,13 @@ A trust policy binds a set of verification configurations against a set of regis
   scopes:
     - "myregistry.io/namespace1/image*"
   keys:
-    - provider: inline-keymanagementprovider-1
+    - provider: cluster-inline-keymanagementprovider-1
 - name: policy-2
   version: 1.0.0  
   scopes:
     - "myregistry.io/namespace2*"
   keys:
-    - provider: inline-keymanagementprovider-2
+    - provider: default/namespaced-inline-keymanagementprovider-2
```

Comment:
+1. I think it might be confusing here to reference a namespaced KMP here without directly explaining to users that this is a multi-tenant scenario. In general, I think all of our individual verifier guides' examples should focus on the cluster scenario. Here I'd be in favor of leaving it referencing a cluster-scoped KMP instead.

Reply:
Thanks for calling it out. I'll leave it as it is. For the walkthrough doc, I feel we can keep it for the default behavior as multi-tenancy is an optional feature. Users could reference this multi-tenancy doc instead.